Specs/Sequence/English/PhoneNumberModel.json

```diff
@@ -1,4 +1,19 @@
 [
+  {
+    "Input": "Tel: 000 111 82-2100.",
+    "Results": [
+      {
+        "Text": "000 111 82-2100",
+        "Start": 5,
+        "End": 19,
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "score": "1",
```

Comment:
You can see the side effect here. This is not a typical number, so it should not have a 1.0 score.
Perhaps SpecialPhoneNumberRegex or somewhere else is a better place to match it. As for the appropriate score, take a look at the other spec cases to see if the score given it to it makes sense compared to others.